sdk/core/azure-core/test/ut/curl_connection_pool_test.cpp

```diff
@@ -41,7 +41,7 @@ namespace Azure { namespace Core { namespace Test {
 
 /***********************  Unique Tests for Libcurl   ********************************/
 #if defined(BUILD_CURL_HTTP_TRANSPORT_ADAPTER)
-
+#if !defined(DISABLE_HTTP_BIN_TESTS)
```

Comment:
```suggestion
#if !defined(_azure_NO_HTTP_BIN_TESTS)
```

1. Prefixed with `_azure_`
2. `DISABLE` makes it looks like it can be set to `0` or `false`, and also it is not really a "disable" - as they are not even compiled if the macro is defined.

Not a strong suggestion, just an opinion.